azDevOps/azure/templates/jobs/build-dotnet.yml

```diff
@@ -133,27 +146,53 @@ jobs:
       inputs:
         command: test
         projects: '${{ parameters.contract_test_project }}'
-        arguments: '-v n'
+        arguments: '--logger trx -v n $(code_coverage_arguments)'
         testRunTitle: 'Test: Provider Contracts'
 
+# Collect code coverage reports into one directory for Sonarqube
+  - ${{ if eq(parameters.static_code_analysis, true) }}:
+    - bash: |
+      'rm -rf ${{ parameters.collected_code_coverage_directory }}'
+      'mkdir -p ${{ parameters.collected_code_coverage_directory }}'
+      'find . -name "${{ parameters.code_coverage_filename }}" -exec cp --backup=numbered {} ${{ parameters.collected_code_coverage_full_path }} \;'
```

Comment:
This might be able to be removed if Sonarqube supports `**` for this. Once I've tested this I'll update accordingly.

Reply:
Taking [this](https://github.com/OmegaGraf/compose/blob/master/.travis.yml) as example, **globbing** should be possible

Reply:
Yeah globbing definitely is possible, the documentation seems to suggest otherwise though!

Reply:
The current version has globbing in there.